deps.edn

```diff
@@ -75,7 +75,7 @@
 
   :lint
   {:extra-deps {clj-kondo/clj-kondo {:mvn/version "2021.09.25"}}
-   :main-opts  ["-m" "clj-kondo.main" "--lint src/clj"]}
+   :main-opts  ["-m" "clj-kondo.main" "--lint" "src/clj"]}
```

Comment:
what happened here :smile:

Reply:
It doesn't work for me unless all the arguments are in different strings